truex-ios-mobile-reference-app/UnlockSoloViewController.m

```diff
@@ -0,0 +1,313 @@
+//
+//  UnlockSoloViewController.m
+//  truex-ios-mobile-reference-app
+//
+//  Created by kyle on 9/27/21.
+//  Copyright © 2021 true[X]. All rights reserved.
+//
+
+#import "UnlockSoloViewController.h"
+#import "WebViewViewController.h"
+#import <TruexAdRenderer/TruexAdRenderer.h>
+
+@interface UnlockSoloViewController ()
+
+@property TruexAdRenderer* activeAdRenderer;
+@property NSMutableDictionary* vastDictionary;
+
+@property (weak, nonatomic) IBOutlet UISwitch *unlocked;
```

Comment:
Does this reset when you reload the view?

Reply:
Yes it does. I mentioned that on the page too.